src/main/java/org/tron/common/runtime/RuntimeImpl.java

```diff
@@ -346,20 +403,29 @@ private void create()
 
       AccountCapsule creator = this.deposit
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move the declaration of "creator" closer to the code that uses it. ([squid:S1941](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1941))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbSdHUuSgN4uCclEWnA&open=AWbSdHUuSgN4uCclEWnA)

[ISSUE_KEY]: AWbSdHUuSgN4uCclEWnA